dashboard/app/models/pd/district_report.rb

```diff
@@ -0,0 +1,59 @@
+module Pd
+  class DistrictReport
```

Comment:
Hmm. Actually I played around a bit more and this behavior is inconsistent. Sometimes I get the wrong class even in the 2nd example. I'll convert it to use full namespaces for class references and remove the module definitions.